pkcs11/pkcs11_test.go

```diff
@@ -35,11 +35,11 @@ import (
 	"testing"
 )
 
-var requireSoftHSMv2 = flag.Bool("require-libsofthsm2", false,
+var requireSoftHSMv2 = flag.Bool("require-libsofthsm2", true,
```

Comment:
For now let's avoid committing this. Instead use the `-require-libsofthsm2` flag from the command line

Reply:
Okay